var/spack/repos/builtin/packages/c-blosc/package.py

```diff
@@ -32,6 +32,8 @@ class CBlosc(CMakePackage):
     depends_on('zstd')
     depends_on('lz4')
 
+    patch('gcc.patch')
```

Comment:
@tkameyama will this patch work on versions other than `1.16.3`? If not, could you add `when='@1.16.3'`?

Reply:
I'm checked version now. And add when=.